resources/views/master.blade.php

```diff
@@ -211,24 +211,17 @@ class="caret"></span></a>
 <script type="text/javascript" src="{{ config('voyager.assets_path') }}/lib/js/toastr.min.js"></script>
 <script>
     @if(Session::has('message'))
-    var type = "{{ Session::get('alert-type', 'info') }}";
-    switch (type) {
-        case 'info':
-            toastr.info("{{ Session::get('message') }}");
-            break;
-
-        case 'warning':
-            toastr.warning("{{ Session::get('message') }}");
-            break;
-
-        case 'success':
-            toastr.success("{{ Session::get('message') }}");
-            break;
-
-        case 'error':
-            toastr.error("{{ Session::get('message') }}");
-            break;
+
+    var alertType = {!! json_encode(Session::get('alert-type', 'info')) !!};
+    var alertMessage = {!! json_encode(Session::get('message')) !!};
+    var alerter = toastr[alertType];
```

Comment:
Wonderful approach, much easier to maintain and read.

However would it be a good idea to split it into another view? Since some of the views easily can become quite big, so about 10 lines of Javascript could go into another view to clean a few things up.